client/src/components/BestNearByExperience.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const NearByExperience = ({ nearByExperience }) => (
+  <div className="nearbyexperience-container-outer">
+    <div className="nearbyexperience-container-inner">
+      <div className="nearbyexperience-header">
+        Get to know the area
+      </div>
+      <div className="experience-icon">
+        <span className="icon-circle">
+          <span className="icon-heart" />
+        </span>
+      </div>
+      <img alt="" className="experience-image" src={nearByExperience.image} />
+      <div className="experience-details-container">
+        <span className="experience-kind">
+          {nearByExperience.kind}
+        </span>
+        <a className="experience-detail-title">
+          {nearByExperience.name}
+        </a>
+        <div className="experience-ratings-container">
+          <span className="experience-rating-bubbles">
+            {nearByExperience.reviewRating}
+          </span>
+          <span className="experience-rating-count">
+            {nearByExperience.reviewCount}
+          </span>
+          <div className="experience-language-container">
+            <img alt="google" src="https://static.tacdn.com/img2/google/powered_by_google_translate.png" />
+          </div>
+          <div className="experience-review-description">
+            {nearByExperience.description}
+          </div>
+          <div className="experience-price-container">
+            <span className="experience-price-amount">
+              {nearByExperience.priceType + nearByExperience.priceNumber}
+              <span className="experience-priceunit">
+                {nearByExperience.priceUnit}
+              </span>
+            </span>
+            <div className="experience-more-info">
+              More info
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+);
+
+NearByExperience.propTypes = {
+  nearByExperience: PropTypes.shape({
```

Comment:
You might be able to use PropTypes.objectOf(PropTypes.oneOfType([PropTypes.string, PropTypes.number]) here to reduce strictness while still passing eslint

Reply:
Excellent, will do! Thanks!